lib/defaults/index.js

```diff
@@ -41,7 +41,7 @@ const defaults = {
 
   transitional: transitionalDefaults,
 
-  adapter: ['xhr', 'http'],
+  adapter: ['xhr', 'http', 'fetch'],
```

Comment:
installed, but at the back, where it will not be selected as a default except on platforms where Axios is not supported w/o shims today (`xhr` will hit on all browsers, `http` will hit on node)